package.json

```diff
@@ -53,13 +53,14 @@
     "expo-asset": "~8.2.1",
     "expo-av": "~8.7.0",
     "expo-camera": "~9.1.0",
+    "expo-cli": "^4.2.1",
```

Comment:
Il me semble qu'on installe expo-cli en global et pas dans le projet.
Je ne sais pas si c'est une bonne pratique (l'avertissement d'expo parle d'un `npm i -g expo-cli`) et expo-cli n'a pas l'air d'être dans nos dépendances.
Qu'en penses-tu ?

Reply:
yep c'est ce qu'il me semblait ahah, je rajoute à la PR

Reply:
Il semblerait que l'installation globale soit conseillée : 
- https://docs.expo.io/get-started/installation/
- https://forums.expo.io/t/should-i-install-expo-locally-or-globally-same-with-rn/42467

Reply:
Du coup est-ce qu'on ne l'enleverait pas du package.json ?

Reply:
hum il est pas dans le package.json normalement

Reply:
C'est etrange je l'ai dans mon package.json sur ta branche. Tu ne l'as pas ?

Reply:
hum nope je l'ai pas, tu es à jour ? fais un `rm -rf node_modules && rm package-lock.json && npm i` pour voir ?